.teamcity/_self/projects/WPComTests.kt

```diff
@@ -330,6 +332,28 @@ fun jetpackBuildType(screenSize: String): BuildType {
 	}
 }
 
+private object GutenbergPlaywrightE2EDesktop: BuildType ({
+	name = "Playwright WPCOM E2E Tests (desktop)"
```

Comment:
/nit I'd drop `WPCOM` from the name, as they are already in the `WPCom Tests` group.

Reply:
I'll incorporate this suggestion in the next revision, thanks!

Reply:
Though, if 'WPCOM' was dropped, wouldn't that also affect how the name is displayed on a PR?

Reply:
![image](https://user-images.githubusercontent.com/975703/131897916-eeb86178-7ba7-42fe-83a6-66c6b0eacd9c.png)

I think it does `${testName} (${group})`. If I'm not mistaken, with the current name it would be `"Playwright WPCOM E2E Tests (desktop) (WPCom Tests)"`